Example/InstanceID/Tests/FIRInstanceIDTokenManagerTest.m

```diff
@@ -56,23 +59,23 @@ - (BOOL)checkTokenRefreshPolicyForIID:(NSString *)IID;
 - (void)updateToAPNSDeviceToken:(NSData *)deviceToken isSandbox:(BOOL)isSandbox;
 /**
  *  Create a fetch operation. This method can be stubbed to return a particular operation instance,
- *  which makes it easier to unit test different behaviors.
+ *  which makes it easier to unit test different behaviours.
```

Comment:
Hmm, an American to Canadian change.  @ryanwilson Do we have any standard guidance about English dialects in comments?

Reply:
I can revert it. I just followed Xcode suggestions